packages/example/src/GifTest/gif-duration.tsx

```diff
@@ -0,0 +1,34 @@
+import {getGifDurationInSeconds} from '@remotion/gif';
+import {useEffect} from 'react';
+import {AbsoluteFill, staticFile} from 'remotion';
+
+const GifTest: React.FC = () => {
+	const giphy = staticFile('giphy.gif');
+
+	useEffect(() => {
+		Promise.all([
+			getGifDurationInSeconds(giphy),
+			getGifDurationInSeconds(
+				'https://media.giphy.com/media/xT0GqH01ZyKwd3aT3G/giphy.gif'
+			),
+			getGifDurationInSeconds(
+				'https://media.giphy.com/media/3o72F7YT6s0EMFI0Za/giphy.gif'
+			),
+		]).then((d) => {
+			console.log('GIF durations', d);
+		});
+	}, [giphy]);
+
+	return (
+		<AbsoluteFill
+			style={{
+				backgroundColor: 'white',
+				fontSize: 100,
+			}}
+		>
+			Check the console!
```

Comment:
I would just add a local state and then reflect the state value here , instead of `check the console`

Reply:
Thanks, just added that with another commit on top of main